patternlab/styleguide/source/_meta/_01-foot.twig

```diff
@@ -33,6 +33,7 @@
       };
     </script>
     <script src="../../assets/js/vendor-generated.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.form/4.2.2/jquery.form.min.js" integrity="sha384-FzT3vTVGXqf7wRfy8k4BiyzvbNfeYjK+frTVqZeNDFl8woCbF0CYG6g2fMEFFo/i" crossorigin="anonymous"></script>
```

Comment:
Should we not keep this consistent and use bower to pull in jquery.form instead of referencing a cdn directly?

Reply:
Also @clair0917, I believe at one point, we talked about not pulling in jquery.form and doing some simplified error messages.  Do you still think that is a good way to go?